web/src/stream_create.js

```diff
@@ -139,6 +139,44 @@ function toggle_advanced_configurations() {
     }
 }
 
+export function toggle_stream_creation_tabs(next_tab) {
+    if (next_tab) {
+        // Hide stream creation containers and show add subscriber container
+        $(
+            ".create_stream_title_container, .create_stream_description_container, .create_stream_types_container",
+        ).hide();
+        $(".stream_create_add_subscriber_container, .finalize_create_stream").show();
+        $("#create_stream_next_tab").hide();
+        $("#create_stream_back_tab").show();
+        $("#subscription_overlay .stream-info-title").text(
+            $t({defaultMessage: "Create channel: add subscribers"}),
+        );
+    } else {
+        // Hide add subscriber container and show stream creation containers
+        $(
+            ".create_stream_title_container, .create_stream_description_container, .create_stream_types_container",
+        ).show();
```

Comment:
We can probably add a containing element and just hide and show one element.

Reply:
Also, probably one change can be we can use `show_subs_pane.create_stream`. Since we are also changing the heading we can consider it as we are changing the complete right panel and that can be handled there. We can probably pass a variable to `show_subs_pane.create_stream` with the name of the container to show and probably a separate function to handle the buttons in footer.